File: Class work/17-8-24/task1.c
#include<stdio.h>

int main()

{
   int a,b,c;
   printf("Enter the value of a: ");
   scanf("%d",&a);
   printf("Enter the value of b: ");
   scanf("%d",&b);
   printf("Enter the value of c: ");
   scanf("%d",&c);

   if(a>b)
   {
    if(a>c)
    {
        printf("%d is the largest number",a);
    }
    else
    {
        printf("%d is the largest number",c);
    }
   }
   else
   {
    if(b>c)
    {
        printf("%d is the largest number",b);
    }
    else
    {
        printf("%d is the largest number",c);
    }
   }
}

File: Class work/Python/17-12-24/task1.py
def fun1():
    n = int(input("Enter Number :"))
    fac = 1

    for i in range(1,n+1):
        fac*=i

    return fac

print(fun1())

def add():
    n = int(input("Enter Number 1 :"))
    n1 = int(input("Enter Number 2 :"))

    return n+n1

print(add())


def fun(n):
    
    fac = 1

    for i in range(1,n+1):
        fac*=i

    return fac

n1 = int(input("Enter Number :"))

print(fun(n1))

def add(n,n1):
    

    return n+n1


a = int(input("Enter Number 1 :"))
b = int(input("Enter Number 2 :"))

print(add(a,b))